### PR TITLE
Submission

### PR DESCRIPTION
**Pull request by odunze**
Please review @kerryknight

### CODE REVIEW

**Comment by kerryknight** [comment]
Hi @odunze, thanks so much for doing this.  Just FYI, I'm trying to get to these as fast as I can, so thanks for your patience.

**Comment by odunze** [comment]
> Hi @odunze, thanks so much for doing this.  Just FYI, I'm trying to get to these as fast as I can, so thanks for your patience. 

Sounds good. Take your time. Ready when you are.

File: Recommendations/View Controllers/RecommendationsViewModel.swift [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+//  Copyright © 2020 Serial Box. All rights reserved.
+//
+
+import UIKit
+
+class RecommendationsViewModel {
+    
+    private let imageCache = NSCache<NSString, UIImage>()
+    
+    var showTopTen: Bool = true
```
Comment (kerryknight) [inline Recommendations/View Controllers/RecommendationsViewModel.swift:15]: `fileprivate(set)`

**Comment by kerryknight** [comment]
Thanks for doing this!  Going to close this PR now.  Please look to hear more from me by mid-week next week.

**Pull request closed without merging**